packages/wallet-backend/src/auth/auth.service.ts

```diff
@@ -39,7 +39,10 @@ export const appendAccessTokenToCookie = (
   expiresIn: number
 ) => {
   res.cookie('AccessToken', token, {
-    httpOnly: true,
+    httpOnly: env.NODE_ENV === 'production',
+    secure: env.NODE_ENV === 'production',
+    sameSite: 'none',
+    domain: env.NODE_ENV === 'production' ? 'rafiki.money' : 'localhost:4003',
```

Comment:
I think domain should be in an env variable, that will allow to setup other environments without changing the code